src/test/unit/test_main_utils.py

```diff
@@ -234,60 +269,35 @@ def test_format_frame():
         "fileName": frame_info.filename,
         "function": frame_info.function,
     }
-    assert variables["a"] == "A"
-    assert variables["password"] == "****"
-
-
-@pytest.mark.parametrize(
-    ["value", "output"],
-    (
-        (
-            {"hello": "world", "inner": {"check": "abc"}},
-            {"hello": "world", "inner": {"check": "abc"}},
-        ),
-        ({"hello": "world", "password": "abc"}, {"hello": "world", "password": "****"}),
-        ({"hello": "world", "secretPassword": "abc"}, {"hello": "world", "secretPassword": "****"}),
-        (
-            {"hello": "world", "inner": {"secretPassword": "abc"}},
-            {"hello": "world", "inner": {"secretPassword": "****"}},
-        ),
-        ('{"hello": "world", "password": "abc"}', '{"hello": "world", "password": "****"}'),
-        (b'{"hello": "world", "password": "abc"}', '{"hello": "world", "password": "****"}'),
-        ('{"hello": "w', '{"hello": "w'),
-        ("5", "5"),
-        ([{"password": 1}, {"a": "b"}], [{"password": "****"}, {"a": "b"}]),
-        ({None: 1}, {None: 1}),
-        ({SKIP_SCRUBBING_KEYS[0]: {"password": 1}}, {SKIP_SCRUBBING_KEYS[0]: {"password": 1}}),
-    ),
-)
-def test_omit_keys(value, output):
-    assert omit_keys(value) == output
+    assert variables["a"] == '"A"'
```

Comment:
The data here was changed, why? Do we support in the backend for this?

Reply:
yes. This is actually more "jsony" correct. Now the BE will get this as a string